src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -308,6 +315,23 @@ class LocalLambdaRunner {
         await command.execute()
 
         if (this.localInvokeArgs.debug) {
```

Comment:
Nit: Suggest to check `if (debugPort)` instead, to avoid the non-null assertion later on.